src/test/java/com/jcabi/aether/MavenClasspathTest.java

```diff
@@ -55,35 +56,76 @@
 public final class MavenClasspathTest {
 
     /**
+     * Group of test artifact.
+     */
+    private static final String GROUP = "junit";
```

Comment:
@amihaiemil so it should be local variable of that test, and second test could have another dependency